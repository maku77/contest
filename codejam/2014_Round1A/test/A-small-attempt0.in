100
1 2
01
11
10 10
0000000111 1000110000 0010111100 1001111011 1111001011 0110010010 1101001100 0011010001 0101100110 1110101101
0111010010 1101111000 0010010001 1011001000 0100000100 0110111111 1100110011 1010101110 1001001111 0001100101
10 10
1110011011 1010010101 0000101010 0111010100 1011110110 0111100011 1000101111 0101011001 1101001000 0000100100
0010011001 1000101000 1001001011 1111110101 0101011110 0111100100 1010010010 0101101001 0010010111 1100100110
2 7
0000001 0111011
0111010 0000000
10 10
0000010100 1000100100 0110111101 0110010000 0101000010 1001101110 1111111011 1001101011 1011001001 0110010111
1011110100 0101101101 0011000100 0101000000 1000011001 1100101011 0110010010 1010111110 1010111011 0101000111
4 6
110000 001011 100001 001010
111100 010110 000111 111101
10 10
1010100001 1100011111 0001000000 0101000111 1001011110 1100111010 0011101001 0110100101 0011111100 1110010010
0011001010 0110010011 0001110110 1001101000 1110100100 0011011111 1010010111 1100001100 1100101001 0101110001
8 9
100111000 110111010 100000101 001000111 001000101 001111010 110000111 001111000
000000101 010111010 101000111 101000101 101111010 000111000 010000111 101111000
6 7
1111001 0000001 1011101 1110110 1100110 0101011
0011100 1001110 1010001 1101010 0110110 1000001
8 10
0010011110 1110101010 0000110110 0101010010 1110011110 0000111011 1111111111 1101100101
0010010001 1011000111 0010101110 1110101010 0011110010 0010101100 1011101011 0000001010
1 10
0001111100
1100111011
3 5
01001 00001 01000
11100 10100 11101
4 7
0100110 1101000 1100000 0101110
1101110 0101000 0100000 1100110
4 10
0000100011 0011000010 0110000111 0110101101
1101000011 0110101001 0110110100 1110000010
2 8
00111111 00111110
01010001 01010000
10 10
1111001101 1011110111 0100010000 0001011010 1001100100 0100010010 0110101001 1001101111 0010000000 1110111111
0100001101 0001000101 1110100000 0011111110 0111010111 1010011010 1100111000 0001000111 1100110011 1011101000
3 9
100110011 000000011 101101000
101001010 100010001 001111010
10 10
1011011000 1110101111 0100110101 0001011011 1110100000 1001011110 1100000001 0111110100 0010000011 0001101110
0000110101 1110110111 1001101110 1011101000 1100010110 0101000010 0110000011 0011011000 0011101101 1100011001
10 10
0110011011 0101010100 1100000001 0000101111 1011111011 1110100110 0011001110 1001100000 0100011100 1011110001
0111001110 0010011011 0100000001 0101001001 1111110011 1000110101 1101010100 1010100100 1010101110 0001111010
10 10
1111101001 1100000110 0000001111 1110111110 0001010011 0001010001 1000010000 0111111000 1111101100 0010100111
0111011010 1001111011 0011001100 0100110101 1010001101 1100100100 1010001111 0100110000 1011010011 0101100010
4 8
00010011 10000111 10101101 00111001
11101100 01010010 11000110 01111000
9 6
000100 000101 101010 011001 011011 100011 001000 100000 101101
011100 011000 110111 010100 100010 110101 001011 111110 000101
1 2
10
11
1 8
11010010
01010010
3 8
11001000 01111010 11101011
00111111 11111101 10010100
2 9
000000001 010100110
000001001 010101110
6 7
0101111 1111101 0111101 0011100 0001110 1011100
0001100 1111111 0011110 0101101 1011110 0111111
9 9
011110010 001111001 110101000 010100110 011010101 001010001 110100101 110101110 101000001
100100011 110001111 100000100 001010011 110100111 101010000 001011000 001011110 010110111
8 8
10100011 00100111 10101111 01011100 00101101 11010010 01010110 11011110
01011110 00100101 01010100 11011100 11010000 10100001 00101111 10101101
9 9
111011110 101010110 010010100 100110010 101110111 101010100 101000010 011011001 101100011
110000000 100001010 000001101 110010110 001000000 110110111 110100011 110000010 111100110
10 10
1001101011 1011100100 0000011111 0100011011 0100100101 0110000001 1001011000 0110001110 1011110010 1111110100
0111001110 1010111101 1000011001 0101010111 0111011000 0011001000 0101100100 1000100111 1010110010 1100100011
8 9
010000100 010010110 111100001 111111011 101000110 000000101 011110111 111011100
001010101 010100001 111011111 111110101 100001011 111010000 011101011 111101100
10 10
1001001110 0011011010 1100101001 0011101110 0001100101 1001111111 1100010001 0110010010 1110100101 0110010000
1011011111 1100110100 0011101010 1001010011 1100000101 0110010100 0100011111 0110100000 0011101000 1001101011
9 10
1101000001 1001001110 0011101001 1110101001 1101111001 0000010000 1100111111 1010111100 0110110010
0010100000 1111001001 1011111110 1100011001 1111110001 1110001111 0100000010 1000001100 0001011001
2 4
0100 1001
1110 0111
10 10
1011100001 1100101100 0010100010 0110001011 1001110001 1011000111 0101011111 1111011100 0100011110 0000110000
1111000100 1001101001 0000101000 0101000111 1011011111 1100110100 0010111010 0100000110 0110010011 1011111001
4 9
100011011 101011011 011110001 011101001
101110111 000110010 011010001 101011100
8 7
0101111 1100101 0110110 1010110 0110101 1110010 1101011 0000110
0001110 0010001 1001000 1000000 0110010 1010100 1100101 0110111
2 8
01000000 01000010
01010111 01000001
4 10
0100110100 1010101101 1000100100 0110111101
0101011011 1011000010 0111010010 1001001011
2 7
0101000 1000000
1100110 0110111
1 10
0001011010
0000001011
3 4
1100 1101 1111
1001 1011 0011
4 9
101000000 010001111 110111000 111001010
100011010 101010000 011101000 001010010
2 10
1101000011 0111101011
0101100111 0011001110
5 6
101101 101010 000100 110101 000101
010010 001011 010000 110110 000010
2 4
0110 0101
0100 0011
4 6
110110 010111 011101 111100
100110 000111 001101 101100
10 10
1111101111 1011000001 0000011010 0010010101 1110100110 0101110000 1100001011 1000110000 0001011111 0111101100
0000011001 0111000110 1000110011 0011101000 1001110110 0100100010 1010111100 1101011001 1111000101 0110001111
8 10
0010011111 1001010010 0010001010 1101110001 0000110011 0110100000 1000101101 1110100110
1011100111 1100100000 0100010111 1111111110 0100011101 1100101110 0011001010 0000010010
8 10
0011011101 1100010011 1110101010 1110111011 0011001100 1100000010 0111001100 0111011101
1000110101 1010001100 0101000010 0101010011 1010011101 0001010011 1000100100 0001000010
5 8
00110100 01000011 11101011 11001101 11101111
01001011 00000000 11000011 10110011 11100010
8 8
11110000 10110111 11111000 10001111 11001000 11010000 10111111 10010111
11001000 11110000 10111111 10010111 10110111 11111000 11010000 10001111
10 10
1111001000 1101010111 1100011101 1010010000 0011110010 1000100101 0110101010 0010110111 0101101010 0001001101
1011100001 1001111110 0111111011 0011011100 1100100110 0100000001 1010101011 1110010011 0101000100 0000011100
6 7
0111011 1000111 0010000 0000001 1011011 1010101
1001010 0111100 0000001 1010110 1001011 1110001
10 10
0001011000 1101011011 1110110101 1000010111 0110011100 0100100001 1011000110 0011101010 0110101111 1001100000
0110011010 0001001111 0000111000 1001110111 1100001110 1011000101 1110000000 1110110011 0011101001 0101110100
9 7
0101010 0001000 1011001 0011111 1000110 1011101 0010011 1001000 0000101
0110101 1101011 0101011 0000110 0001101 1000101 1110110 0010010 0010100
4 8
11000000 01000011 11000001 01000010
11011001 01011010 01011011 11011000
8 10
1111100000 1101001011 1111001011 1010100000 1110100000 1110001011 1010001011 1101100000
0000011111 0001110100 0000110100 0001011111 0010110100 0101110100 0010011111 0101011111
10 10
1110001010 0000001010 0101011101 1101011111 0010111100 1110100101 0101110001 1001100111 1010100010 0011010000
0111001001 1110111011 0001101000 1101111110 1010010011 0100010011 0110100101 1001000110 0001000100 1010111100
2 10
0001111101 0101111100
1001101001 1101101000
10 7
0110111 0111010 1101101 0000111 0000001 1100000 0100110 1001100 0101111 0101001
0000010 0001111 1101110 1000110 1011000 0100011 1010101 1001001 1101000 1000000
10 10
1111010111 0100110010 1010101000 1010101111 0000010001 0100001100 1001111101 1011101100 0111000010 0101010011
0101101110 1011001010 1010001110 0100110001 0110100000 0001110011 1011001101 1110110101 1000011111 0101010000
10 10
0110000110 0011110000 0000101001 1000110010 0111100011 0001000010 0100101011 1111110011 0001100010 0000000111
1010000000 0110111011 1011110111 0101011101 1010101000 1001010001 1010110100 1000110001 1111110010 1100001111
10 10
1101000000 0000111001 1010010011 1010011100 0101100110 1101110001 0010101111 1110001011 0111111110 0001000100
0010011111 0001010100 1000100001 1110011011 1101110000 0010101110 0101000011 0101001100 1111100110 1010111001
6 5
01001 01011 01100 11001 11011 11100
00101 10010 00111 10101 00010 10111
4 7
1010011 1111001 0111100 0010110
1110001 0011110 0110100 1011011
8 5
10010 11110 01111 00011 10110 01001 00111 11000
11100 10110 10010 01011 00111 01101 11010 00011
10 10
1100000010 0011110111 0111000010 0000101000 1111101101 1000110111 0100001101 1001011100 0111110000 1010011011
1111000000 0110101100 1011000111 0011011010 1100011111 0101101011 0000110101 1000111010 0100000000 1011110101
10 10
0111100000 1101111011 1010100011 1000000110 0110111111 0110010110 1100001000 0001101101 1011011100 0001010001
0010111100 0100010111 1111100101 1000001011 1111011001 0011001111 1001010100 1000100010 0101101000 0110110010
10 10
1111001000 0000000100 1000100000 1001110011 0111110111 1100011001 1110011101 0011111110 0110100110 0001001011
0100001001 0011100001 0111011000 1101100111 1100110110 0101011100 1010001010 1011000101 0010110010 1000111111
6 6
101111 110010 010011 000001 110001 100111
101011 000010 100100 000100 111011 101111
10 10
0011010111 1101111001 1100001010 1010111100 0100101100 0110100010 1111000101 0010010001 0001010110 1001101011
1011111110 1111001101 1100001010 1001110000 0100110111 0010011001 0001010110 0111100000 0000100101 1110001011
7 9
010010010 011111111 111111111 010111000 111110010 011000011 101111000
000000010 100001111 100000010 000111110 001101111 110000101 001000101
10 10
0110010000 0000111111 0100000001 1001001110 1010001110 0011110001 1111010110 0111101011 1001110001 1100101100
0000101010 0101110111 1010010110 0110001000 0011010000 1111110111 0101001000 1000000111 1011101101 1100111001
2 10
1111110111 1101110100
0101011000 0111011011
10 10
0011100000 1110010111 1101000100 0110100001 1001011000 1101111111 0100011010 0000110000 1011001111 0010101111
1110111010 0111011111 1100000001 0011111000 0111100100 1010010000 0001101111 1001000000 1000001111 0100110111
5 7
0010110 1111001 1011000 1011011 1110111
1110100 1011000 0010101 1011011 1111010
9 8
01010111 00110101 10010110 11011101 10100001 00001110 10101101 00110111 10110011
00101011 11010001 01011011 00010000 10110011 00100111 10001000 10110001 00110101
3 5
10100 00111 10000
00111 00110 10000
2 9
010010011 101000010
101100010 010110011
2 7
0101110 0101010
1110000 0111001
10 10
0100010001 1101000011 0010100010 1111111110 0011001100 0101111101 0010101011 1001111110 1100010000 1010000101
0011101111 1101011101 1100001110 0010000001 0100110000 1011001000 1110110011 0011100110 0101011100 1000110011
5 4
1001 0001 1111 0110 1101
0001 0011 0111 1000 1111
10 10
1100001000 1111001001 0010100101 1001101100 0110010011 1101111010 0011010011 1111100110 0000010100 0000111111
1110011100 1100101101 0011110000 1010101010 0101010101 0000110001 0011011111 1111101010 1100000110 0001000011
10 10
0100100001 0101111111 0001111010 1010010111 1011100010 0110011100 1000000001 1011001110 1101110100 0110001001
0000110000 1110101101 0000100101 0010001101 0011010011 1100111011 1101100010 1101001110 1011011000 0111010110
8 5
01101 01100 10011 11010 00101 00100 10010 11011
00101 10011 00100 01101 01100 10010 11010 11011
10 10
0101110101 1011110110 1000001110 1101000011 0000001101 0110111011 1001110100 0110000000 0110011011 1011101000
1010000111 1010011100 0100010010 0101101000 1100010001 0001011111 1010100111 1001101001 0111101010 0111110100
6 5
11111 11100 01011 01000 01111 01100
11111 11011 11000 01100 11100 01111
10 5
00011 10100 00101 10010 01010 11101 00110 10111 11011 00111
00111 01000 01110 11100 11111 00001 11001 10000 01101 11101
10 10
1011101010 1000000100 0100100001 0001011111 0101010100 0100000000 1110111011 1010101011 1111110100 0011011111
1110100000 1010111110 1011110001 1101001110 0000011110 0001101011 0100010101 1111100001 0110010101 0001001010
2 6
101010 101011
111101 111100
10 10
1111000010 0000100100 1110111010 0011110110 1100011001 0001110111 0110101011 1101001100 0011010001 1000001101
1000010110 0110100011 1111001010 0111011011 0101111000 0100101101 0001101100 1010110000 1010010111 1001000101
10 10
1000000111 1100011001 0011111110 0101011100 1110010001 0001100001 1010010000 1011100111 0101101010 0110101110
1010101111 1110100100 1000110000 0011001001 0111010111 0101011111 1110010010 0001011110 1101100000 0000101001
8 10
1011001000 0101111111 0011001101 0000110011 0110100011 1011110000 0011111010 1100110100
1111010000 1100001100 1001100010 1001010101 0001011111 1010011100 0001100111 0110011011
10 10
0101111111 0010001111 0111101100 0001001110 1110010110 1010010000 1100100000 0001110011 1100110001 1011001001
1101110101 1100101100 1011010100 1001110011 1011000101 0000001001 0101101010 0110101011 0010011010 0110010110
8 4
0111 0100 1101 0110 0010 0000 0011 1001
0001 1010 0000 0011 0101 0111 1110 0100
10 10
0111110101 1111100110 1000000001 0101110011 1100001010 0011111111 1001011000 0010001100 0100101011 1010010100
0111011110 1000101010 0011010101 1101010011 0000111001 1010101100 1011110100 0101001011 0110000111 1100100000
6 5
00000 01111 00100 01110 00101 01010
10100 11110 10000 11011 10001 11010
6 5
11000 01011 11010 01001 11001 11011
11001 01000 01001 01010 11011 01011
