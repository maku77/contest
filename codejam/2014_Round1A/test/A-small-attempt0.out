Case #1: 1
Case #2: 3
Case #3: 6
Case #4: 1
Case #5: 3
Case #6: 5
Case #7: 4
Case #8: 1
Case #9: 5
Case #10: NOT POSSIBLE
Case #11: 6
Case #12: 3
Case #13: 1
Case #14: NOT POSSIBLE
Case #15: 5
Case #16: 6
Case #17: 5
Case #18: 5
Case #19: 4
Case #20: 6
Case #21: 2
Case #22: NOT POSSIBLE
Case #23: 1
Case #24: 1
Case #25: NOT POSSIBLE
Case #26: 1
Case #27: 1
Case #28: 7
Case #29: 1
Case #30: 4
Case #31: 6
Case #32: NOT POSSIBLE
Case #33: 6
Case #34: 3
Case #35: NOT POSSIBLE
Case #36: 2
Case #37: NOT POSSIBLE
Case #38: NOT POSSIBLE
Case #39: NOT POSSIBLE
Case #40: 5
Case #41: NOT POSSIBLE
Case #42: 3
Case #43: NOT POSSIBLE
Case #44: NOT POSSIBLE
Case #45: NOT POSSIBLE
Case #46: NOT POSSIBLE
Case #47: NOT POSSIBLE
Case #48: 1
Case #49: 4
Case #50: NOT POSSIBLE
Case #51: 5
Case #52: NOT POSSIBLE
Case #53: 0
Case #54: 6
Case #55: NOT POSSIBLE
Case #56: 6
Case #57: NOT POSSIBLE
Case #58: 2
Case #59: 6
Case #60: 4
Case #61: 3
Case #62: 6
Case #63: 3
Case #64: NOT POSSIBLE
Case #65: 9
Case #66: 3
Case #67: 1
Case #68: 1
Case #69: 7
Case #70: 6
Case #71: 4
Case #72: NOT POSSIBLE
Case #73: 6
Case #74: 7
Case #75: 4
Case #76: 4
Case #77: 3
Case #78: 2
Case #79: 3
Case #80: NOT POSSIBLE
Case #81: 1
Case #82: NOT POSSIBLE
Case #83: 4
Case #84: 3
Case #85: 6
Case #86: 5
Case #87: 0
Case #88: 5
Case #89: 1
Case #90: 3
Case #91: 4
Case #92: 3
Case #93: 4
Case #94: 6
Case #95: 7
Case #96: 6
Case #97: 2
Case #98: 9
Case #99: 2
Case #100: 1
